honeybadger-io/honeybadger-js
# #648 Callback not called on lambda handler if handler is async

## Status
**READY**

## Description
This fixes another issue described in this [ticket](https://github.com/honeybadger-io/honeybadger-js/issues/648#issuecomment-951378899):
When passing async handlers to `Honeybadger.lambdaHandler`, the callback function is never called.

## Related PRs
n/a

## Todos

## Steps to Test or Reproduce
1. Create an async lambda handler (`handler`):
```
const handler = client.lambdaHandler(async function(_event, _context) {
    return Promise.resolve({message:'works!'})
})
```
2. Call `handler` and expect a result:
```
handler({}, {}, (err, res) => {
    expect(err).toBeFalsy()
    expect(res).toBeDefined()
    expect(res.message).toEqual('works!')
    done()
})
```
